datamade/committee-oversight
# Add project logo

## Overview

Closes #120. This PR adds a new project logo from the client, which has gone through some back and forth. It should look like this:

<img width="1351" alt="Screen Shot 2019-12-16 at 12 05 54 PM" src="https://user-images.githubusercontent.com/1094243/70931290-b8bf8400-1ffc-11ea-8173-cfd94c9d2d7c.png">

On small screens, we should just see the Lugar logo.

This PR also fixes the vulnerability described [here](https://github.com/datamade/committee-oversight/network/alert/requirements.txt/django/open) by fixing Django at 2.1.15.

## Testing Instructions

* Run the site following the instructions in the README
* Look at it at various sizes and on various pages and make sure the topnav looks good.
